src/components/Directory/MentorCard/MentorCard.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, cleanup, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import MentorCard from './MentorCard';
+import { jssPreset } from '@material-ui/styles';
+
+// describe accepts 2 arguments a string and a function
+// MentorCard should have a Mentors name/ icon/ and bio
+// when running test your specific file npm run test <filename>
+// each testing needs its own rendered item/ set up the enviornment so that that section can be tested start with a render
+// than you can set up expectations of what you want to see.
+// check that each of these properties exist and show up in the card
+// check if the button can be clicked and is visible
+afterEach(cleanup);
```

Comment:
no longer needed in latest version of RTL

Reply:
the cleanUp afterEach function?